src/overlays/actors/ovl_En_Zf/z_en_zf.c

```diff
@@ -452,9 +452,13 @@ s16 EnZf_FindNextPlatformAwayFromPlayer(Vec3f* pos, s16 curPlatform, s16 arg2, G
         }
     }
 
-    // These functions have no side effects, so these two calls do nothing
+    //! @bug `altNextPlatform` can be -1 in certain conditions and cause an out of bounds access.
+    //! Under normal conditions, this doesn't cause problems because the data before `sPlatformPositions`
+    //! is section padding between .text and .data, so 0 gets read as a float.
+#ifndef AVOID_UB
     Math_Vec3f_DistXYZ(&player->actor.world.pos, &sPlatformPositions[nextPlatform]);
     Math_Vec3f_DistXYZ(&player->actor.world.pos, &sPlatformPositions[altNextPlatform]);
+#endif
```

Comment:
did you mean to get rid of the comment on this code being essentially useless

Reply:
I did mean to get rid of the comment that speaks directly to the AVOID_UB stuff, but yeah I can add that other part of it back to the overall bug explanation